### ISSUE TITLE
SpaceAroundKeyword warns about chaining methods on yield

### ISSUE THREAD

**Issue by maxjacobson** [bbatsov/rubocop#2775]
```ruby
def some_method
  puts yield.reverse
end

some_method { 'oliM' }
```

With RuboCop 0.37 this produces this warning:

```
Inspecting 1 file
C

Offenses:

test.rb:2:8: C: Style/SpaceAroundKeyword: Space after keyword yield is missing.
  puts yield.reverse
       ^^^^^

1 file inspected, 1 offense detected
```

yield is a keyword, but it returns a value and maybe you'll want to chain a method on it. I can move the method call to the next line but it doesn't feel like an ideal change
